aspnetcore/fundamentals/localization.md

```diff
@@ -96,7 +96,7 @@ DataAnnotations error messages are localized with `IStringLocalizer<T>`. Using t
 
 [!code-csharp[Main](localization/sample/ViewModels/Account/RegisterViewModel.cs?start=9&end=26)]
 
-The runtime doesn't look up localized strings for non-validation attributes. In the code above, "Email" (from `[Display(Name = "Email")]`) will not be localized.
+ASP.NET Core 1.0 doesn't look up localized strings for non-validation attributes. In the code above, "Email" (from `[Display(Name = "Email")]`) will not be localized. In ASP.NET Core 1.1.0 and higer, non-validation attributes are localized.
```

Comment:
Should be `MVC Core 1.0` and `MVC Core 1.1.0` since this is MVC behavior and not ASP.NET, right? Also, and this is kind of a nit-pick, I would put the fact that it IS localized in 1.1.0 before the fact that it isn't in 1.0 since hopefully people will be using 1.1.0 much more than 1.0 in the future.